packages/core/src/utils/feature-flag.ts

```diff
@@ -0,0 +1,28 @@
+import crypto from 'node:crypto';
+
+type Properties = {
+  /**
+   * The unique identifier of the feature flag entity.
+   */
+  entityId: string;
+  /**
+   * The percentage of requests that should be have the feature flag enabled.
+   * The value should be between 0 and 1.
+   */
+  rollOutPercentage: number;
+};
+
+/**
+ * Check if a request is included in an A/B test.
+ * We use the entityId to determine if feature flag should be enabled for the request.
+ */
+export const isFeatureFlagEnabled = ({ entityId, rollOutPercentage }: Properties) => {
```

Comment:
the comments and the function name is confusing

Reply:
Should remove all the A/B test phrases.  Miss this.

Reply:
Updated